src/gui/properties/propertieswidget.ui

```diff
@@ -1017,14 +1017,14 @@
          <item>
           <widget class="QPushButton" name="selectAllButton">
            <property name="text">
-            <string>Select All</string>
+            <string>Check all items</string>
```

Comment:
I don't like verb "Check" here. "To check" is not what a user wants to do. In other words, ticking mark in a checkbox is just a mean to do a real action. You don't name, for example, a button "Increase tab index by one", you name it "Go to the next tab". In this case texts like "Download all files" or "Download nothing" explain real actions.